### PR TITLE
Fix: Update dependencies in wire-ios-request-strategy

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Fix the compile error due to change of method `addParticipantIfMissing` from https://github.com/wireapp/wire-ios-data-model/pull/768

### CODE REVIEW

File: WireRequestStrategy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1078,10 +1078,9 @@
 			};
 			buildConfigurationList = 166901641D707509000FE4AF /* Build configuration list for PBXProject "WireRequestStrategy" */;
 			compatibilityVersion = "Xcode 3.2";
-			developmentRegion = English;
```
Comment (typfel) [inline WireRequestStrategy.xcodeproj/project.pbxproj]: Revert

Comment (billypchan) [reply WireRequestStrategy.xcodeproj/project.pbxproj]: It can be included in this PR, to fix the warning raised from XCode 10.2

**Comment by typfel** [comment]
Tests also fail to build?

**Comment by billypchan** [comment]
@zenkins release patch

**Pull request merged**